xbmc/guilib/GraphicContext.cpp

```diff
@@ -983,16 +984,17 @@ void CGraphicContext::SetMediaDir(const std::string &strMediaDir)
   m_strMediaDir = strMediaDir;
 }
 
+
+void CGraphicContext::SetStereoMode(RENDER_STEREO_MODE mode)
+{
+  m_stereoMode = mode;
+  g_application.m_pPlayer->TriggerUpdateResolution();
```

Comment:
I see and this is what I usually call crrrrr :)
This is the same no-go. gfx context must not call into upper layers and WindowManager in in an upper layer.